index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+   
+    <div class="flex-container">
+    <header>
+        <nav class="navbar">
+            <ul>
+                <li class="nav-logo">My Logo</li>
+                
+            </ul>
+        </nav>
+    </header>
```

Comment:
- [ ] There should be a hamburger button on the right side which is missing in your design. 
<img width="216" alt="Capture" src="https://user-images.githubusercontent.com/58504655/120799403-209ebd00-c558-11eb-9027-0af384372022.PNG">